airflow-core/src/airflow/ui/src/hooks/usePluginTabs.tsx

```
@@ -31,36 +31,22 @@ type TabPlugin = {
-    if (colorMode === "dark" && view.icon_dark_mode !== undefined && view.icon_dark_mode !== null) {
-      iconSrc = view.icon_dark_mode;
-    }
-
-    const icon =
-      iconSrc !== undefined && iconSrc !== null ? (
-        <img alt={view.name} src={iconSrc} style={{ height: "1rem", width: "1rem" }} />
-      ) : (
-        <LuPlug />
-      );
+    // an icon can be provided for menu items, but for now we will just use the default
+    const icon = <LuPlug />;
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)